src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentLoaderService.cs

```diff
@@ -20,11 +20,15 @@ namespace Microsoft.CodeAnalysis.PdbSourceDocument
     internal sealed class PdbSourceDocumentLoaderService : IPdbSourceDocumentLoaderService
     {
         private const int SourceLinkTimeout = 1000;
-        private readonly ISourceLinkService? _sourceLinkService;
+
+        /// <summary>
+        /// Lazy import ISourceLinkService because it can cause debugger 
```

Comment:
I think you forgot to

Reply:
it's like we're here to start eachothers...